pennylane/tape/qubit_param_shift.py

```diff
@@ -29,6 +29,56 @@
 from .jacobian_tape import JacobianTape
 
 
+def _get_operation_recipe(op, p_idx, shifts):
+    """Utility function to return the parameter-shift rule

```

Comment:
@dwierichs was this copied here to keep `QubitParamShiftTape` working? Next week I am going to add deprecation warnings to all the tape subclasses, and the week after, will delete them :)

Reply:
That makes me very happy :) 
It was copied here essentially to avoid circular imports (but also conceptually `QubitParamShiftTape` depending on the `gradients` module felt a bit strange). I thought it is okay to do it like this because of the planned deletion :)